Rubberduck.Inspections/Abstract/InspectionBase.cs

```diff
@@ -58,7 +58,7 @@ protected InspectionBase(RubberduckParserState state, CodeInspectionSeverity def
         /// <summary>
         /// Meta-information about why an inspection exists.
         /// </summary>
-        public virtual string Meta => InspectionsUI.ResourceManager.GetString(Name + "InspectionMeta", CultureInfo.CurrentUICulture);
+        public virtual string Meta => InspectionsUI.ResourceManager.GetString(Name + "Meta", CultureInfo.CurrentUICulture);
```

Comment:
I already mentioned in my PR that it was a monkey-patch ... this may have broken the display in the settings window ... Did you check that?